variables.tf

```diff
@@ -314,6 +314,12 @@ variable "runners_request_spot_instance" {
   default     = true
 }
 
+variable "runners_check_interval" {
+  description = "defines the interval length, in seconds, between new jobs check."
+  type        = number
+  default     = 0
```

Comment:
Ok by me, other option could be to set the default to 3 https://docs.gitlab.com/runner/configuration/advanced-configuration.html